functions/analyzing_functions.R

```diff
@@ -1,4 +1,4 @@
-source("functions/helper_functions.R")
+source("helper_functions.R")
```

Comment:
So funktioniert das bei mir nicht mehr, die vorherige Version müsste schon richtig sein

Reply:
@PhilippSL, dann kannst du in Console die Ordner festlegen, zum Beispiel:

`setwd("~/GitHub/GitHub-Projekt-Wissenschaftliches-Arbeiten/functions")`

Sonst muss jeder das Programm jedes Mal modifizieren.

Reply:
@danfleon Es muss bei `source("functions/helper_functions.R")` bleiben. Da alle Pfade die verwendet werden von der Hauptseite des Reporitory ausgehen sollten. 
Wenn man dann einmal `setwd(.../GitHub-Projekt-Wissenschaftliches-Arbeiten)` setzt, ist der gesamte Code für jeden lauffähig.

Reply:
Ja, es stimmt @jonasmolsbeck. Sorry @PhilippSL.